### ISSUE TITLE
Anki addon downloads fail with SSL_CERTIFICATE_VERIFY_FAILED

### ISSUE THREAD

**Issue by Pneumaticat** [NixOS/nixpkgs#45726]
## Issue description

With the current packaged version of Anki, downloading addons fails with an SSL_CERTIFICATE_VERIFY_FAILED.

### Steps to reproduce

Download any addon from the Ankiweb library. I used [Hierarchical Tags](https://ankiweb.net/shared/info/1089921461), but any addon causes the issue.

Go to `Tools → Add-ons → Browse & Install` to enter the ID. Then this error shows up in a dialog box:

```
Download failed: Traceback (most recent call last):
  File "/nix/store/ni000mrx2yqsj2f60y6azjgd7vq41f5i-anki-2.0.52/lib/python2.7/site-packages/aqt/downloader.py", line 67, in run
    aqt.appShared + "download/%d" % self.code)
  File "/nix/store/4i2ig2z3lg3rkx1k68yz9gphlcngh0g6-python2.7-httplib2-0.11.3/lib/python2.7/site-packages/httplib2/__init__.py", line 1694, in request
    (response, content) = self._request(conn, authority, uri, request_uri, method, body, headers, redirections, cachekey)
  File "/nix/store/4i2ig2z3lg3rkx1k68yz9gphlcngh0g6-python2.7-httplib2-0.11.3/lib/python2.7/site-packages/httplib2/__init__.py", line 1487, in _request
    redirections=redirections - 1)
  File "/nix/store/4i2ig2z3lg3rkx1k68yz9gphlcngh0g6-python2.7-httplib2-0.11.3/lib/python2.7/site-packages/httplib2/__init__.py", line 1694, in request
    (response, content) = self._request(conn, authority, uri, request_uri, method, body, headers, redirections, cachekey)
  File "/nix/store/4i2ig2z3lg3rkx1k68yz9gphlcngh0g6-python2.7-httplib2-0.11.3/lib/python2.7/site-packages/httplib2/__init__.py", line 1434, in _request
    (response, content) = self._conn_request(conn, request_uri, method, body, headers)
  File "/nix/store/ni000mrx2yqsj2f60y6azjgd7vq41f5i-anki-2.0.52/lib/python2.7/site-packages/aqt/sync.py", line 467, in _conn_request
    conn.connect()
  File "/nix/store/4i2ig2z3lg3rkx1k68yz9gphlcngh0g6-python2.7-httplib2-0.11.3/lib/python2.7/site-packages/httplib2/__init__.py", line 1127, in connect
    raise SSLHandshakeError(e)
SSLHandshakeError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:726)
```

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.4, NixOS, 18.09.git.498ad50 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0.4`
 - channels(root): `""`
 - nixpkgs: `/etc/nixos/nixpkgs`

**Comment by Pneumaticat** [comment]
The update to anki 2.1 (#46599) will close this issue, according to my testing. (Add-on downloads work perfectly.)

**Comment by Pneumaticat** [comment]
(oops, I'll keep it open until that PR is merged)

**Comment by w3rs** [comment]
Given that some useful addons don't support Anki 2.1, I'd like this to be fixed for 2.0. OTOH I'm not sure how this can be done.

**Comment by mkaito** [comment]
You can still manually install addons by dropping the files in `~/.local/share/Anki2/addons`.

An example is described [here](https://github.com/glutanimate/anki-addons-misc#miscellaneous-anki-add-ons).
